jarbas/api/tests/test_reimbursement_view.py

```diff
@@ -1,59 +1,19 @@
-from datetime import date
 from json import loads
 from unittest.mock import patch
+from urllib.parse import urlencode
 
-from django.core.cache import cache
 from django.shortcuts import resolve_url
 from django.test import TestCase
 
+from mixer.backend.django import mixer
+
 from jarbas.core.models import Reimbursement
-from jarbas.core.tests import sample_reimbursement_data, suspicions
```

Comment:
If we're replacing this fixture, we can cleanup `jarbas/core/tests/__init__.py` and use `mixer` in other tests that depends on them… using `mixer` _sometimes_ IMHO tends to lead to a bigger mess ; )